### PR TITLE
Update benchmarks CI to use less steps to save time

### PR DESCRIPTION
**Pull request by conectado**
SSIA

### CODE REVIEW

File: scripts/generate-weights.sh [hunk 1]
```diff
@@ -62,6 +65,5 @@ benchmark council
 benchmark referendum
 benchmark forum
 benchmark membership
-benchmark bounty
```
Comment (shamil-gadelshin) [inline scripts/generate-weights.sh]: Why did you remove the bounty pallet?

**Review by shamil-gadelshin — CHANGES_REQUESTED** [review]
Looks good. But it looks like you removed the bounty pallet from benchmarking.

File: scripts/generate-weights.sh (hunk 1 above)
Comment (conectado) [reply scripts/generate-weights.sh]: Sorry, a mess up I made with git

**Review by shamil-gadelshin — APPROVED** [review]
